src/opm/parser/eclipse/EclipseState/Schedule/ScheduleState.cpp

```diff
@@ -65,17 +65,26 @@ const PAvg& ScheduleState::pavg() const {
     return *this->m_pavg;
 }
 
+void ScheduleState::nupcol(int nupcol) {
+    this->m_nupcol = nupcol;
+}
+
+int ScheduleState::nupcol() const {
+    return this->m_nupcol;
+}
 
 bool ScheduleState::operator==(const ScheduleState& other) const {
     return this->m_start_time == other.m_start_time &&
            this->m_tuning == other.m_tuning &&
            this->m_end_time   == other.m_end_time;
+           this->m_nupcol == other.m_nupcol;
```

Comment:
This statement has no effect.  You need to add an `&&` operator on the previous line too.

Reply:
Rebase gone bad - fixed